Not sure what's going on.
When I try to build this PR as is, I get:
```
error: build of '/nix/store/rhvcnixr5f4sbm684bzy12d3pcsjn8zg-ir.lv2-0-unstable-2018-06-21.drv' on 'ssh://nixBuild@builder' failed: builder for '/nix/store/rhvcnixr5f4sbm684bzy12d3pcsjn8zg-ir.lv2-0-unstable-2018-06-21.drv' failed with exit code 2;
       last 10 log lines:
       >                  from /nix/store/vdaqdd9l32mnpxlv645y4sjslm82yv9x-glib-2.78.3-dev/include/glib-2.0/gobject/gbinding.h:30:
       > /nix/store/vdaqdd9l32mnpxlv645y4sjslm82yv9x-glib-2.78.3-dev/include/glib-2.0/glib/gtypes.h:578:26: note: declared here
       >   578 | typedef struct _GTimeVal GTimeVal GLIB_DEPRECATED_TYPE_IN_2_62_FOR(GDateTime);
       >       |                          ^~~~~~~~
       > ir_gui.cc:1659:21: error: invalid conversion from 'void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*)' {aka 'void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)'} to 'void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*)' {aka 'void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)'} [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-fpermissive-fpermissive8;;]
       >  1659 |         {IR_UI_URI, instantiate, cleanup, port_event, NULL}
       >       |                     ^~~~~~~~~~~
       >       |                     |
       >       |                     void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}
       > make: *** [Makefile:43: ir_gui.o] Error 1
```
       
I'm pretty sure it built without issues before.

--- a/pkgs/applications/audio/ir.lv2/default.nix
+++ b/pkgs/applications/audio/ir.lv2/default.nix
@@ -14,6 +14,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [  pkg-config ];
 
+  env.NIX_CFLAGS_COMPILE = "-fpermissive";
+
   postBuild = "make convert4chan";
 
   installPhase = ''

